netlify.toml

```diff
@@ -1,2 +1,9 @@
 [build]
   functions = "functions"
+
+# Temporary build command for preview
+[context."update-redirects-location"]
+  command = "npm run build"
+
+[context."test-redirect-change-build"]
+  command = "npm run build"
```

Comment:
What does this mean, "temporary build command?" Does this mean it should be removed before merging?

Reply:
Correct, I have a TODO in the description for this. Our build command in Netlify is `make build`. Since we're removing the Makefile, I needed to setup alternative build commands for these branches.